src/web/components/KeyPairs/KeyPairsTable.scss

```diff
@@ -4,6 +4,7 @@
   display: flex;
   flex-direction: column;
   align-items: stretch;
+  margin-bottom: 75px;
```

Comment:
This gap feels a bit big for me. I'm also planning on changing how we space components generally in https://github.com/IABTechLab/uid2-self-serve-portal/pull/369. Perhaps once we are consistent with where all the "Add" buttons are positioned (i.e. above the table, rather than below), this will not cause as much confusion, and hence a smaller gap would be acceptable?

Reply:
Sounds good, reduced to 50px.  I think while you were out, we decided all add buttons should be above the table so that it does not become hard to find once someone has a lot on their table and has to scroll.  A smaller gap is good, but lets not keep it so small that it could look like Add Domain belongs to the key pair table - that was a concern Kimberly brought up I believe